sidedish/src/component/ItemDetail/DetailModal.jsx

```diff
@@ -0,0 +1,253 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { StyledBadges } from 'component/DishItem/DishItem';
+import Counter from 'component/ItemDetail/Counter';
+import RecommendItem from 'component/ItemDetail/RecommendItem';
+import DetailCarousel from './DetailCarousel/DetailCarousel';
+
+const DetailModal = ({ detailData, loading, title, badge }) => {
+  const [count, setCount] = useState(1);
+  const [topImg, setTopImg] = useState(detailData.top_image);
+  const thumbImgList = detailData.thumb_images.map((img, i) => {
+    return (
+      <img className="thumb_img" key={i} src={img} alt={i} onClick={() => setTopImg(img)}></img>
+    );
+  });
```

Comment:
p5) implicit return을 사용하시면 가독성이 조금 좋아진다고 하네요
```suggestion
  const thumbImgList = detailData.thumb_images.map((img, i) => 
      <img className="thumb_img" key={i} src={img} alt={i} onClick={() => setTopImg(img)} />  });
```
https://betterprogramming.pub/21-best-practices-for-a-clean-react-project-df788a682fb

Reply:
return 을 명시하고 싶어서 return()을 항상 고집했었는데,
좀 더 클린한 코드를 위해서는 앞으로 implicit return 을 의식적으로 쓰려고 노력해야겠습니다.
감사합니다!